plugins/plugins_test.go

```diff
@@ -340,6 +340,34 @@ func TestPluginManagerConsoleLogger(t *testing.T) {
 	}
 }
 
+func TestPluginManagerServerInitialized(t *testing.T) {
```

Comment:
Nit: It would be great if you could add a comment as to what you're trying to test in the 2 test cases below. It's not clear to me at least.

Reply:
Thanks, I somehow missed you weren't calling `ServerInitialized()` in the second case.

Reply:
Oh, I definitely was calling it lol, and it was passing because I was erroneously using `return` above it instead of `break`.  So good catch 😁